mkt/ratings/forms.py

```diff
@@ -47,11 +45,11 @@ def save(self):
                                          is_flagged=is_flagged))
                     if self.request:
                         ReviewerScore.award_moderation_points(
-                            self.request.user, addon, review_id)
+                            self.request.user, addon, review.id)
                 elif action == REVIEW_MODERATE_KEEP:
                     review.editorreview = False
                     review.save()
-                    mkt.log(mkt.LOG.APPROVE_REVIEW, review.addon, review,
+                    mkt.log(mkt.LOG.APPROVE_REVIEW, addon, review,
```

Comment:
oh its `review` not `review.id` here as well, so I guess either both are right or wrong